gossip/simulation/gossip.go

```diff
@@ -15,6 +15,8 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
+// +build ignore
```

Comment:
So it won't be built into the simulation package but can still be built by doing `go run gossip.go`. I could move it to its own separate package, but that didn't seem worth it.